src/client/pages/CubeDraftPage.tsx

```diff
@@ -68,31 +67,6 @@ const processPredictions = (json: PredictResponse, packCards: any[]) => {
   return packCards.map((card) => predictionsMap.get(card.oracle_id) || 0);
 };
 
-const getInitialState = (draft: Draft): State => {
```

Comment:
Moved to draftutils and added tests. Ideally later we can do this in the route and pass the initial state direct to the page.